docs/conf.py

```diff
@@ -1,3 +1,7 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
Suspicious of this! We're debugging in Real Life™, so let's keep working on that.

Reply:
I'd probably vote for removing this from here, and opening it in a separate PR, perhaps also addressing https://github.com/showyourwork/showyourwork/issues/312 from @rwegener2. What do you think of that?

Reply:
I have a theory here! I predict that @katiebreivik and @afeinstein20 have edited their `$PYTHONPATH`'s by setting them like `export PYTHONPATH=/some/path` in their shell rc or profile file. The `$PYTHONPATH` has to have a blank entry in it for Python to include the current directory in the search path (i.e. in `sys.path`). 

I modify my `$PYTHONPATH` locally, but I do it like:
```bash
export PYTHONPATH=$PYTHONPATH:/some/path
```
The default `$PYTHONPATH` is a blank space, so this is equivalent to:
```bash
export PYTHONPATH=:/some/path
```

With this, if I do
```bash
python -c "import sys; print(sys.path)"
```
in a path, I see that path in the list - e.g., in my home dir:
```
['', '/Users/apricewhelan', '/Users/apricewhelan/anaconda3/lib/python310.zip', ... more crap ...]
```

But without the empty entry in the `$PYTHONPATH`, my home dir disappears from the list.

Reply:
So! interestingly, I haven't actually modified the `$PYTHONPATH`, _but_ when I do
`python -c "import sys; print(sys.path)"`
I don't have the current directory on my path; so regardless of the directory, I get:
`['', '/Users/kbreivik/opt/anaconda3/envs/cosmic/lib/python310.zip', ...]`
So I have the empty entry, but not the current directory.... 

@afeinstein20 : is this also the case for you?